scania-digital-design-system/sdds-website
# AB#251 Solves issue with missing bootstrap styling

- Bootstrap added to website
- Created a file to import missing old style until new is done with the components





**Describe pull-request**  
_Describe what the pull-request is about_
Issue with missing styling of bootstrap is removed from SDDS
Until we have created components that are used some styling needs to be imported from bootstrap
- Anchor
- Navigation in content
- Modal both style and JS
- Tables
- Checkboxes pictogram
- Inputfield pictogram

**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
A part of scania-digital-design-system/sdds#51

**How to test**  
_Add description how to test if possible_
- Run site with npm link with theme and components
- See if any styling is missing in the list

**Screenshots**  
_If applicable, add screenshots to help explain_

**Additional context**  
_Add any other context about the pull-request here._
